### PR TITLE
Unplanned work for routing

### PR DESCRIPTION
**Pull request by Taracuda**
This PR covers routing for existing pages and includes a new HomePageComponent, which will be what appears when a user first visits CollectR. It contains "Login" and "Browse" buttons, with "Login" routing to the login page. The "Browse" button will not function now until the search page has been created.

### CODE REVIEW

**Comment by conhur** [comment]
Looks good. I can see the change handler on line 19.

File: src/components/HomePageComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export const HomeComponent: React.FC = () => {
+  return (
+    <>
+      <img src={CollectRImage} className="CollectR-logo" alt="CollectR-logo" />
+      <div>
+        <button>
+          <Link to={"/login"}>Login</Link>
+        </button>
+        {/* //Browse button does not do anything upon click yet */}
```
Comment (piperlincoln) [inline src/components/HomePageComponent.tsx:13]: When it is clicked in the future, will it bring the user to the search page?

**Review by piperlincoln — APPROVED** [review]
Looks perfect Tara, thank you!

**Review by firicano33 — COMMENTED** [review]
Hi Tara.  The add of unplanned code for routing to each individual page looks good.

File: src/components/HomePageComponent.tsx (hunk 1 above)
Comment (Taracuda) [reply src/components/HomePageComponent.tsx:13]: Yes, that's correct!

**Comment by hsantos940** [comment]
Everything looks good to me.

**Pull request merged**